types/xlsx/index.d.ts

```diff
@@ -173,13 +173,147 @@ export interface IWorkBook {
     Props: IProperties;
 }
 
+export interface IColInfo {
+    /**
+     * Excel's "Max Digit Width" unit, always integral
+     */
+    MDW?: number;
+    /**
+     * width in Excel's "Max Digit Width", width*256 is integral
+     */
+    width: number;
```

Comment:
Notice the comment about the priority order:
> 1) use width field if available 2) use wpx pixel width if available 3) use wch character count if available

Also, I've been using it with only `wch` and it works fine. So `width` is not required.